Add Mosviz Jwebbinar Demo Notebook

duytnguyendtn opened the pull request:
Here's my notebook for the Jwebbinar on JDAT's Mosviz, and a requirements file!

eteq reviewed (commented):
Can you rename the requirements to `requirements.txt`?  That's consistent with what the other webbinars are doing.  (it'll require merging with the other PRs in this directory, but there's no helping that if we want the users to get a one-shot install step :Shrug:)

eteq commented on jdat_session/MosvizJwebbinarDemoRequirements.txt:
```diff
@@ -0,0 +1,3 @@
+decorator==4.4.2
+astropy==4.2.1
+jdaviz==1.1
```
```suggestion
decorator>=4.4.2
astropy>=4.2.1
jdaviz>=1.1
```
I suggest schanging this to "lower pinnings", because I suspect most users are not going to be wanting to have a stand-alone webbinar environment.,

duytnguyendtn replied:
Hmm interesting; I imagined we would be asking users to start in a fresh environment for reproducibility purposes. If users are bringing in their own environments, couldn't that be a massive headache if things break?

camipacifici replied:
We will not support their own environments for the webinar. They will all work on the AWS platform.

duytnguyendtn replied:
Actually, if that's the case, should we be testing our notebooks against a specific environment @camipacifici?

camipacifici commented:
Workflow looks good, but this notebook will need more introduction and explanation, links to the documentation, and explanation of the dataset. You can get inspiration from the style in these notebooks: https://github.com/spacetelescope/jwebbinar_prep/tree/main/pipeline_products_session

eteq commented on .gitignore:
```diff
@@ -1,2 +1,5 @@
+*.ipynb_checkpoints*
+
+# Environments
+env*/
```
I was just about to add this in `main` directly, but maybe just include it here so that you don't have to fix the conflit?
```suggestion
env*/

*.fits
```

duytnguyendtn replied:
Works for me!

duytnguyendtn commented:
I added some more description, but half of the activity is embedded within the actual viewer itself, so I found it difficult to add descriptions of the activity. But I added context where I could; let me know if you all have any additional recommendations!

camipacifici commented:
@duytnguyendtn 
Looks better, thank you for adding the documentation!
A few more suggestions:
- You can spell out the tasks you will perform in Mosviz like Patrick does in his Cubeviz notebook.
- Please, provide also a "live" notebook that you will use to code during the webinar. You can find an example here: https://github.com/spacetelescope/jwebbinar_prep/blob/main/pipeline_products_session/jwst-data-products-part1-live.ipynb
